filebeat/input/redis/harvester.go

```diff
@@ -57,6 +57,7 @@ type log struct {
 	cmd       string
 	key       string
 	args      []string
```

Comment:
Are you sure that IP and port is not present in args? Did you check it (could you post a sample log)?